backport_packages/refactor_backport_packages.py

```diff
@@ -0,0 +1,786 @@
+                if any([f"airflow.providers.google.cloud.{class_type}.bigquery" in text
+                        for class_type in CLASS_TYPES]) or "_to_bigquery" in text:
+                    print(f"Ignoring {file_path}")
+                    ignored_names.append(file_name)
+        return ignored_names
+
+    def ignore_kubernetes_files(src: str, names: List[str]) -> List[str]:
+        ignored_names = []
+        if src.endswith(os.path.sep + "example_dags"):
+            for file_name in names:
+                if "example_kubernetes" in file_name:
+                    ignored_names.append(file_name)
+        return ignored_names
+
+    def ignore_some_files(src: str, names: List[str]) -> List[str]:
+        ignored_list = ignore_bigquery_files(src=src, names=names)
+        ignored_list.extend(ignore_kubernetes_files(src=src, names=names))
+        return ignored_list
+
+    rm_build_dir()
+    package_providers_dir = get_target_providers_folder()
+    if os.path.isdir(package_providers_dir):
+        rmtree(package_providers_dir)
+    copytree(get_source_providers_folder(), get_target_providers_folder(), ignore=ignore_some_files)
+
+
+def copy_helper_py_file(target_file_path: str) -> None:
+    """
+    Copies. airflow/utils/helper.py to a new location within provider package
+
+    The helper has two methods (chain, cross_downstream) that are moved from the original helper to
+    'airflow.models.baseoperator'. so in 1.10 they should reimport the original 'airflow.utils.helper'
+    methods. Those deprecated methods use importe with import_string("<IMPORT>") so it is easier to
+    replace them as strings rather than with Bowler
+
+    :param target_file_path: target path name for the helpers.py
+    """
+
+    source_helper_file_path = os.path.join(get_source_airflow_folder(), "airflow", "utils", "helpers.py")
+
+    with open(source_helper_file_path, "rt") as in_file:
+        with open(target_file_path, "wt") as out_file:
+            for line in in_file:
+                out_file.write(line.replace('airflow.models.baseoperator', 'airflow.utils.helpers'))
+
+
+class RefactorBackportPackages:
+    """
+    Refactors the code of providers, so that it works in 1.10.
+
+    """
+
+    def __init__(self):
+        self.qry = Query()
+
+    def remove_class(self, class_name) -> None:
+        """
+        Removes class altogether. Example diff generated:
+
+
+        .. code-block:: diff
+
+            --- ./airflow/providers/google/cloud/operators/kubernetes_engine.py
+            +++ ./airflow/providers/google/cloud/operators/kubernetes_engine.py
+            @@ -179,86 +179,3 @@
+            -
+            -class GKEStartPodOperator(KubernetesPodOperator):
+            -
+            - ...
+
+        :param class_name: name to remove
+        """
+        # noinspection PyUnusedLocal
+        def _remover(node: LN, capture: Capture, filename: Filename) -> None:
+            if node.type not in (300, 311):  # remove only definition
```

Comment:
Minor point: does Bowler expose constants for these? Or if not, can you at add a comment saying what these two types are.

Reply:
I just double checked it - it was there before done by Tomek, but I see that I only get one node type and no 300 nor 311. Also this operator now is restricted to single package by filter, so risk is very small that something wrong will be removed (and we will notice anyway at package preparation phase and in imports/readme generation.  If something gets removed what should not be the safety net we have now with importing all classes and generating README files will show us immediately that something was wrong. 

Removed those nodes.